pyproject.toml

```diff
@@ -2,6 +2,49 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "pybind11~=2.6",
+    "pybind11==2.12.0",
+    "tomli>=0.10; python_version<'3.11'",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "mapbox_earcut"
+version = "1.0.2"
+requires-python = ">=3.7"
+authors = [{name = "Samuel Kogler", email = "samuel.kogler@gmail.com"}]
+license =  {file = "LICENSE.md"}
+description = "Python bindings for the mapbox earcut C++ polygon triangulation library."
+dependencies = ["numpy>=1.19.0"]
```

Comment:
maybe > 1.24? -- not entirely sure how far back the 2.0 compatibility goes -- but no on e needs an ancient numpy :-)